models/collection.go

```diff
@@ -311,11 +311,14 @@ func (c *CollectionApi) Create(ctx context.Context) (*CollectionApi, error) {
 	return c, nil
 }
 
-func (c *CollectionApi) Read(ctx context.Context) (*CollectionApi, error) {
+func (c *CollectionApi) Read(ctx context.Context, ignore404 ...bool) (*CollectionApi, error) {
```

Comment:
that's how I knew that there are no optional parameters in go

Reply:
yep, this is ugly